recipe/run_test.py

```diff
@@ -31,7 +33,6 @@
 import _csv
 import _ctypes
 import _ctypes_test
-import _decimal
```

Comment:
The lack of _decimal will cause the decimal module in the standard library to use the slower Python implementation. Adding `--with-system-libmpdec=no` will restore the C implementation (not sure what the Windows equivalent is)

Reply:
Using libmpdec conda package now